app/main/helpers/search_summary_manifest.yml

```diff
@@ -85,6 +85,28 @@
   filtersPreposition:
   filterRules:
   conjunction: and
+
+
+-
+  id: Supplier type
+  labelPreposition: of type
```

Comment:
This preposition is really weird for the associated filter labels, especially when contrasted against the staff security clearance ones: '272 results found in Cloud hosting of type Reseller providing extra features and support and with staff security clearance that Conforms to BS7858:2012'.

Also the capitalisation on the filter labels is weird; if we're trying to construct proper sentences, we should probably have the filter labels be lower-case and then capitalise appropriately in the templates where they stand alone (e.g. in the filter boxes).